pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1151,14 +1165,25 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
               this.progressTracker.action('container engine components', 50, async() => {
                 await this.writeFile('/etc/containerd/config.toml', CONTAINERD_CONFIG);
                 await this.writeConf('containerd', { log_owner: 'root' });
-                await this.writeFile('/etc/init.d/docker', SERVICE_SCRIPT_DOCKERD, { permissions: 0o755 });
+                await this.writeFile('/usr/local/bin/nerdctl', NERDCTL, 0o755);
+                await this.writeFile('/etc/init.d/docker', SERVICE_SCRIPT_DOCKERD, 0o755);
                 await this.writeConf('docker', {
                   WSL_HELPER_BINARY: await this.getWSLHelperPath(),
                   LOG_DIR:           logPath,
                 });
-                await this.writeFile(`/etc/init.d/buildkitd`, SERVICE_BUILDKITD_INIT, { permissions: 0o755 });
+                await this.writeFile(`/etc/init.d/buildkitd`, SERVICE_BUILDKITD_INIT, 0o755);
                 await this.writeFile(`/etc/conf.d/buildkitd`, SERVICE_BUILDKITD_CONF);
               }),
+              this.progressTracker.action('Openresty configuration', 50, async() => {
```

Comment:
Same as above… also why is the word order reversed here?

Reply:
> why is the word order reversed here?

Probably because it was written days apart?

Done